lib/Jsrt/JsrtDebugManager.cpp

```diff
@@ -146,14 +146,26 @@ HRESULT JsrtDebugManager::DbgRegisterFunction(Js::ScriptContext* scriptContext,
 
     if (!utf8SourceInfo->GetIsLibraryCode() && !utf8SourceInfo->HasDebugDocument())
     {
+        JsrtDebugEventObject debugEventObject(scriptContext);
+        Js::DynamicObject* eventDataObject = debugEventObject.GetEventDataObject();
+
+        JsrtDebugUtils::AddFileNameOrScriptTypeToObject(eventDataObject, utf8SourceInfo);
+        JsrtDebugUtils::AddLineCountToObject(eventDataObject, utf8SourceInfo);
+        JsrtDebugUtils::AddPropertyToObject(eventDataObject, JsrtDebugPropertyId::sourceLength, utf8SourceInfo->GetCchLength(), utf8SourceInfo->GetScriptContext());
+        
```

Comment:
I see this operations in order and also appear on the other functions in this file. Could you refactor to one and call in the other functions as well?